src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -3,5 +3,12 @@
     <OutputPath>$(RuntimeBinDir)ilc/</OutputPath>
     <RuntimeIdentifier>$(OutputRid)</RuntimeIdentifier>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NETStandard.Library">
```

Comment:
Should this be in `ILCompiler.props` instead so that it gets picked up by all build flavors?